API.md

```diff
@@ -460,7 +454,7 @@ When the server contains exactly one connection, `registrations` is an object wh
 registered plugin name and value contains:
 - `version` - the plugin version.
 - `name` - the plugin name.
-- `options` - options used to register the plugin.
+- `options` - options object passed to the plugin during registration.
```

Comment:
If you pass `{ once: true }` to the options does that get passed to the plugin? and does it end up in the options here?